native/src/components/NavigationTiles.tsx

```diff
@@ -34,7 +33,7 @@ const StyledIcon = styled(Icon)<{ disabled: boolean }>`
 `
 
 type NavigationTilesProps = {
-  tiles: Array<TileModel>
+  tiles: TileModel[]
```

Comment:
:thought_balloon: Maybe we can add a linting rule or something, so we stay consistent with either of them.